documentation/src/main/asciidoc/user_guide/integrations_spring.adoc

```diff
@@ -152,14 +195,19 @@ or by adding them to your `hotrod-client.properties` (for client/server uses onl
 </bean>
 ----
 
-==== Externalizing session using Spring Session
+[[spring_externalize_sessions]]
+==== Externalizing Sessions Using Spring Session
 
-link:http://docs.spring.io/spring-session/docs/current/reference/html5[Spring Session] is a very convenient way to externalize user session into {brandname} cluster.
+link:http://docs.spring.io/spring-session/docs/current/reference/html5[Spring Session] lets you externalize user session information into {brandname}.
 
-Spring Session integration allows to use both - embedded and client/server mode. Each mode requires using proper artifacts (`infinispan-spring4-embedded` or `infinispan-spring4-remote`).
-An example is shown below:
+To configure Spring Session integration in your application, do the following:
 
-[source,xml]
+. Add dependencies to your `pom.xml`.
++
+* Embedded mode: `infinispan-spring4-embedded`
+* Remote client-server mode: `infinispan-spring4-remote`
++
+.pom.xml (embedded mode)
```

Comment:
I know it was there before, but there is a bug in the pom.xml

it should be infinispan version

```xml
 <dependency>
             <groupId>org.infinispan</groupId>
             <artifactId>infinispan-spring4-embedded</artifactId>
             <version>${version.infinispan}</version>
         </dependency>
```

Reply:
And I don't know why this ```<artifactId>infinispan-embedded</artifactId>``` is there ...

@tristantarrant any idea ?

Reply:
for me should be "infinispan-core" or even nothing because the embedded jar is bringing core transitively

Reply:
I checked the simple tutorials, and this seems an error. So core instead of embedded